app/javascript/react/src/containers/NavBar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { BrowserRouter as Router, Link } from 'react-router-dom'
+// import BackButton from './BackButton'
```

Comment:
Will you use the BackButton later? Just double checking as we don't want this to be left in accidentally.